src/main/java/com/google/finapp/FinAppService.java

```diff
@@ -39,7 +40,7 @@ public void createCustomer(Customer customer, StreamObserver<Empty> responseObse
           UuidConverter.getBytesFromUuid(UUID.randomUUID()),
           customer.getName(),
           customer.getAddress());
-    } catch (SpannerException e) {
+    } catch (SQLException e) {
```

Comment:
I think you need to wait for Bianca's change ? Otherwise, this'll be a breaking change.

Reply:
Yes, this PR will be edited and merged after Bianca's change.